modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/PrepareMasterDatabaseForRelease.pm

```diff
@@ -236,30 +236,31 @@ sub pipeline_analyses_prep_master_db_for_release {
                 'db_conn'     => '#master_db#',
                 'input_query' => 'UPDATE method_link_species_set SET url = "" WHERE source = "ensembl"',
             },
-            -flow_into  => [ 'hc_master' ],
+            -flow_into  => [ 'dc_master' ],
         },
 
-        {   -logic_name      => 'hc_master',
-            -module          => 'Bio::EnsEMBL::Compara::RunnableDB::RunJavaHealthCheck',
+        {   -logic_name      => 'dc_master',
+            -module          => 'Bio::EnsEMBL::Compara::RunnableDB::RunDataChecks',
             -parameters      => {
-                'compara_db'  => '#master_db#',
-                'work_dir'    => $self->o('work_dir'),
-                'testgroup'   => 'ComparaMaster',
-                'output_file' => '#work_dir#/healthcheck.#testgroup#.out',
-                'ensj_conf'   => $self->o('ensj_conf'),
-                'run_healthchecks_exe' => $self->o('run_healthchecks_exe'),
-                'ensj_testrunner_exe'  => $self->o('ensj_testrunner_exe'),
+                'datacheck_groups' => ['compara_master'],
+                'work_dir'         => $self->o('work_dir'),
+                'history_file'     => '#work_dir#/datacheck.compara_master.history.json',
+                'output_file'      => '#work_dir#/datacheck.compara_master.tap.txt',
+                'failures_fatal'   => 1,
+                'datacheck_types'  => ['critical'],
+                'registry_file'    => $self->o('reg_conf'),
+                'compara_db'       => '#master_db#',
+            },
+            -flow_into      => {
+                1 => { 'backup_master' => { 'output_file' => $self->o('master_backup_file') } },
             },
-            -flow_into       => [ 'backup_master' ],
-            -rc_name         => '2Gb_job',
             -max_retry_count => 0,
         },
 
         {   -logic_name => 'backup_master',
             -module     => 'Bio::EnsEMBL::Hive::RunnableDB::DatabaseDumper',
             -parameters => {
                 'src_db_conn' => '#master_db#',
-                'output_file' => $self->o('master_backup_file'),
```

Comment:
Why have you removed this line ?

Reply:
Because the ensembl-datachecks RunDataChecks runnable flows the `output_file` parameter down branch 1 to the `backup_master` analysis and override the same named parameter. I didn't know how else to do it, ideas welcome :shrug:

Reply:
Ah, I see ! I missed the other change.
So yes, what you did is one option. The other is to keep the `output_file` as an analysis parameter like it was and make the dataflow like this:
```
1 => { 'backup_master' => {} }
```
to restore the empty input_id.
I don't have any preferences

Reply:
Actually, I think I would prefer leaving it as an analysis parameter, like in the other backup analysis (`backup_current_master`) for consistency